.circleci/config.yml

```
@@ -12,32 +12,36 @@ jobs:
       - checkout
 
       - run:
-          name: Install micromamba
+          name: Install MiniForge
           shell: /bin/bash -l
           command: |
             apt update --yes && apt-get upgrade --yes
             apt install -y --no-install-recommends wget ca-certificates git
 
             cd ${HOME}
-            curl -Ls https://micro.mamba.pm/api/micromamba/linux-64/latest | tar -xvj bin/micromamba
-            eval "$(${HOME}/bin/micromamba shell hook -s posix)"
+            wget -O  Miniforge3.sh "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
+            bash Miniforge3.sh -b -p "${HOME}/conda"
+            source "${HOME}/conda/etc/profile.d/conda.sh"
+            source "${HOME}/conda/etc/profile.d/mamba.sh"
 
       - run:
           name: Create RAiDER environment
-          shell: /bin/bash -l
+          shell: /bin/bash -xl
           command: |
-            eval "$(${HOME}/bin/micromamba shell hook -s posix)"
+            source "${HOME}/conda/etc/profile.d/conda.sh"
+            source "${HOME}/conda/etc/profile.d/mamba.sh"
           
             PYTHON_VERSION="<< parameters.python-version >>"
             sed -i "/python>=/c\ - python=${PYTHON_VERSION}" environment.yml
-            micromamba create -f environment.yml
+            mamba env create -f environment.yml
 
       - run:
           name: Install raider and check environment
-          shell: /bin/bash -l
+          shell: /bin/bash -xl
           command: |
-            eval "$(${HOME}/bin/micromamba shell hook -s posix)"
-            micromamba activate RAiDER
+            source "${HOME}/conda/etc/profile.d/conda.sh"
+            source "${HOME}/conda/etc/profile.d/mamba.
```

```suggestion
            source "${HOME}/conda/etc/profile.d/mamba.sh"
```

This should get you off and running again